### ISSUE TITLE
[Suggestion] Double door operation for Forestry doors

### ISSUE THREAD

**Issue by xenoflot** [VazkiiModsArchive/Charset#32]
Hi,

I hope this is the right place. Is Charset the mod in FTB Unstable responsible for double door operations with vanilla doors?

If so, it would be great if the operation could be extended to include Forestry doors in addition to vanilla doors.

Thanks!

**Comment by palaster** [comment]
Yes, charset add double doors and it is possible to add that ability to mod doors, BUT I don't know what would happen if a door that wasn't meant to be open normally responds to double door operations, i.e. iron door.

**Comment by xenoflot** [comment]
Anticipating your diligent caution, Mezz over at Forestry offered to discuss it.
https://github.com/ForestryMC/ForestryMC/issues/1489

**Comment by palaster** [comment]
Technically this is not my call, but I have a solution add mod doors to double doors.

**Comment by xenoflot** [comment]
A quick googling doesn't show up any double door mods for 1.10 at this time.

**Comment by palaster** [comment]
Sorry I misspoke, the function in Charset that add that feature is called double doors. I have a solution available at #33.

**Comment by Dushess** [comment]
Note: same function exist in MalisisDoors. 
In addition i have suggestion to add locks on the doors.
